### PR TITLE
Fix entrypoint banner for SDK container

### PR DESCRIPTION
**Pull request by rmccorm4**

### CODE REVIEW

File: Dockerfile.sdk [hunk 1]
```diff
@@ -168,5 +168,13 @@ ARG TRITON_MODEL_ANALYZER_REPO_TAG
 ARG TRITON_MODEL_ANALYZER_REPO="https://github.com/triton-inference-server/model_analyzer@${TRITON_MODEL_ANALYZER_REPO_TAG}"
 RUN pip3 install "git+${TRITON_MODEL_ANALYZER_REPO}"
 
+# Entrypoint Banner
+ARG NVIDIA_TRITON_INFERENCE_SERVER_SDK_VERSION=21.10
```
Comment (deadeyegoodwin) [inline Dockerfile.sdk:172]: We don't want to hardwire this verision. Can you instead get it from the TRITON_VERSION file?

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: Dockerfile.sdk [hunk 2]
```diff
@@ -168,5 +168,13 @@ ARG TRITON_MODEL_ANALYZER_REPO_TAG
...
+ARG NVIDIA_BUILD_ID=unknown
```
Comment (CoderHam) [inline Dockerfile.sdk:173]: Is the convention to use 'unknown' as the default build id?

File: Dockerfile.sdk (hunk 1 above)
Comment (rmccorm4) [reply Dockerfile.sdk]: Will be passed in from CI, changed 21.10 default to `unknown`

File: Dockerfile.sdk (hunk 2 above)
Comment (rmccorm4) [reply Dockerfile.sdk:173]: I took `unknown` as the default from our `build.py` convention: https://github.com/triton-inference-server/server/blob/400cc0cf7287de8f19c227442f1b98f5abe86e55/build.py#L959

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**